internal/instanceid/instanceid.go

```diff
@@ -6,17 +6,21 @@ import (
 	"github.com/google/uuid"
 )
 
+func Make() string {
+	return uuid.New().String()
+}
+
 var (
-	id string
-	mu sync.Mutex
+	appID string
+	mu    sync.Mutex
 )
 
-func ID() string {
+func AppID() string {
```

Comment:
Nit: this feels unnecessary

Reply:
Since I added a construction func, I wanted to ensure the meaning of this function was more clear since it is acting as an app global singleton.

Reply:
I'd say if anything this is more confusing though. It's not an "App ID", that sounds like it would be constant for any state tool invocation.